### PR TITLE
Update the CF Gradle plugin version.

### PR DESCRIPTION
**Pull request by smillst**

### CODE REVIEW

File: checker/build.gradle [hunk 1]
```diff
@@ -221,6 +221,7 @@ clean {
 }
 
 clean.doLast {
+  buildDir.deleteDir()
```
Comment (kelloggm) [inline checker/build.gradle:224]: This change should probably be in a separate PR

**Review by kelloggm — CHANGES_REQUESTED** [review]

File: checker/build.gradle (hunk 1 above)
Comment (smillst) [reply checker/build.gradle]: Fixed. Thanks!

**Review by kelloggm — APPROVED** [review]
